string conductivity
string pH
string temperature
